src/shape/custom_shapes.js

```diff
@@ -1625,26 +2058,76 @@ function customShapes(p5, fn) {
   };
 
   /**
+   * Sets the property of a curve. For example, set tightness,
+   * use `splineProperty('tightness', t)`, with `t` between 0 and 1,
+   * at 0 as default.
+   * 
+   * Spline curves are like cables that are attached to a set of points.
+   * Adjusting tightness adjusts how tightly the cable is
+   * attached to the points. The parameter, tightness, determines
+   * how the curve fits to the vertex points. By default,
+   * tightness is set to 0. Setting tightness to 1, as in
+   * `splineProperty('tightness', 1)`, connects the curve's points
+   * using straight lines. Values in the range from –5 to 5
+   * deform curves while leaving them recognizable.
+   * 
    * @method splineProperty
    * @param {String} property
-   * @returns value The current value for the given property.
+   * @param value Value to set the given property to.
+   * 
+   * @example
+   * <div>
+   * <code>
+   * // Move the mouse left and right to see the curve change.
+   * 
+   * function setup() {
+   *   createCanvas(100, 100);
+   *   describe('A black curve forms a sideways U shape. The curve deforms as the user moves the mouse from left to right');
+   * }
+   * 
+   * function draw() {
+   *   background(200);
+   * 
+   *   // Set the curve's tightness using the mouse.
+   *   let t = map(mouseX, 0, 100, -5, 5, true);
+   *   splineProperty('tightness', t);
+   * 
+   *   // Draw the curve.
+   *   noFill();
+   *   beginShape();
+   *   splineVertex(10, 26);
+   *   splineVertex(10, 26);
+   *   splineVertex(83, 24);
+   *   splineVertex(83, 61);
+   *   splineVertex(25, 65);
+   *   splineVertex(25, 65);
+   *   endShape();
+   * }
+   * </code>
+   * </div>
    */
   /**
    * @method splineProperty
    * @param {String} property
-   * @param value Value to set the given property to.
+   * @returns value The current value for the given property.
```

Comment:
I think for `@returns` we don't need a variable name:
```suggestion
   * @returns The current value for the given property.
```